packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx

```diff
@@ -69,7 +69,7 @@ const Headline = styled.div`
     font-size: 16px;
     font-weight: 600;
     margin-top: 20px;
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
I think it works same with `normal` as I proposed, or not?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break `break-word` is deprecated

Reply:
Ahh, I see, well `work-break: normal` is the default, so I think removing the `word-break` entry entirely is the best approach?